src/main/webapp/WEB-INF/views/car/all_cars.jsp

```diff
@@ -0,0 +1,32 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" %>
+<html>
+<head>
+    <title>All Cars</title>
+    <link rel="stylesheet" href="<c:url value="/static/css/style.css"/>" type="text/css">
+</head>
+<body>
+<h1>All cars</h1>
+    <table>
+        <tr>
+            <th>Car id</th>
+            <th>Model</th>
+            <th>Manufacturer</th>
+            <th>Drivers</th>
+            <th>Add driver</th>
+            <th>Delete car</th>
+        </tr>
+        <c:forEach var="car" items="${cars}">
+            <tr>
+                <td>"${car.id}"</td>
+                <td>"${car.model}"</td>
+                <td>"${car.manufacturer.name}"</td>
+                <td>"${car.drivers}"</td>
```

Comment:
why did you write `<td>"${car.id}"</td>` with `""`?
we don't need it, but if you did it on purpose, then you can leave it as it is